README.md

```diff
@@ -167,6 +167,32 @@ ansible-playbook deploy.yml                                        \
     --user       $user
 ```
 
+### Migrate or reset the database
+
+Control and storage clusters have separate databases that need to be migrated individually.
+In some case, we can also reset database to clear all table information.
```

Comment:
This should not be in this pull request. It's probably coming from the branch below. Please rebase.

Reply:
Done.